tests/foreman/installer/test_installer.py

```diff
@@ -593,25 +588,6 @@ def test_positive_check_installer_hammer_ping(target_sat):
             assert 'ok' in line
 
 
-@pytest.mark.upgrade
-@pytest.mark.tier1
-@pytest.mark.build_sanity
-def test_installer_modules_check(target_sat):
```

Comment:
This change is completely unrelated to ipv6 and http proxy setups.
Would you please split it into separate PR to make life easier during cherrypicking.

This test lives only in master and 6.16.z so you would definitely hit conflicts in older branches. Thanks.

Reply:
I was not planning to cherry-pick it to 6.16.z? Should we?